src/SlnGen.Build.Tasks/build/SlnGen.targets

```diff
@@ -24,6 +24,7 @@
       CollectStats="$([MSBuild]::ValueOrDefault('$(SlnGenCollectStats)', 'false'))"
       CustomProjectTypeGuids="@(SlnGenCustomProjectTypeGuid)"
       DevEnvFullPath="$(SlnGenDevEnvFullPath)"
+      FlatHierarchy="$([MSBuild]::ValueOrDefault('$(SlnGenFlatHierarchy)', 'false'))"
```

Comment:
I'd like a different name for this parameter and property.  I'm worried Hierarchy is too technical.  Maybe something simpler like `NoFolders` or `Flatten`?  

```xml
NoFolders="$([MSBuild]::ValueOrDefault('$(SlnGenNoFolders)', 'false'))"
```
```xml
Flatten="$([MSBuild]::ValueOrDefault('$(SlnGenFlatten)', 'false'))"
```

I can't quite figure out what is the simplest term, ideas?

Reply:
Maybe `CreateFolders`/`CreateFolderStructure` (the opposite) or `IgnoreFolders`/`IgnoreFolderStructure`?

Reply:
Let's go with just `Folders`:

```
Folders="$([MSBuild]::ValueOrDefault('$(SlnGenFolders)', 'true'))"
```

Defaults to `true`.  Also can you update the documentation?  https://github.com/jeffkl/SlnGen/blob/master/docs/README.md  
Replace the last row of the table (SlnGenCollectStats) under **Launching of Visual Studio** with this new property.